lib/internal/test_runner/mock.js

```diff
@@ -157,17 +161,10 @@ class MockTracker {
         'options.setter', setter, "cannot be used with 'options.getter'"
       );
     }
+    validateField(object, methodName);
 
-    const descriptor = ObjectGetOwnPropertyDescriptor(object, methodName);
-    let original;
-
-    if (getter) {
-      original = descriptor?.get;
-    } else if (setter) {
-      original = descriptor?.set;
-    } else {
-      original = descriptor?.value;
-    }
+    const descriptor = findMethodOnPrototypeChain(object, methodName);
+    const original = getOriginalObject({ descriptor, getter, setter });
```

Comment:
No need to make a throw away object here just to destructure it in `getOriginalObject()` - just pass three separate parameters.